opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -420,9 +421,8 @@ def __new__(
             trace_state = DEFAULT_TRACE_STATE
 
         is_valid = (
-            trace_id != INVALID_TRACE_ID
-            and span_id != INVALID_SPAN_ID
-            and trace_id < _TRACE_ID_HEX_LENGTH
+            0 < trace_id <= _TRACE_ID_MAX_VALUE
```

Comment:
Personally I think using the `INVALID_TRACE_ID` reads better, the constant lends itself well to be reused for situations like these.

Reply:
adapted the check to use the constants